js/gutenberg/ss-gutenberg.js

```diff
@@ -89,9 +93,14 @@ const withInspectorControls = createHigherOrderComponent((BlockEdit) => {
 
 
         let simpleShopGroupsData = ssGutenbergVariables.groups;
-        let simpleShopGroups = [{label: __('Doesn\'t matter', 'simpleshop-cz'), value: ''}];
+        let groups = [{label: __('Doesn\'t matter', 'simpleshop-cz'), value: ''}];
         for (let item in simpleShopGroupsData) {
-            simpleShopGroups.push({label: simpleShopGroupsData[item], value: item});
+            groups.push({label: simpleShopGroupsData[item], value: item});
+        }
+
+        let selectedGroups = simpleShopGroups;
+        if (typeof simpleShopGroups === 'undefined' && simpleShopGroup) {
```

Comment:
Tady to vypadá, že `simpleShopGroups` je zřejmě již přejmenováno na `groups`, ale tady se stále používá. Není to chyba?

Reply:
Skupiny ze SS obsahuje promenna `groups`. `simpleShopGroups` teď obsahuje skupiny vybran0 uživatelem. Tahle podmínka říká, že pokud je `simpleShopGroups` undefined (uživatel ještě nezvolil žádné skupiny v novém nastavení) a zároveň existuje `simpleShopGroup` (to je hodnota z původního selectu), je hodnota `simpleShopGroups` pole s tou jednou původní skupinou, podle mě to sedí.